diff --git a/data.cpp b/data.cpp
index 4594dbf..8cb77c9 100644
--- a/data.cpp
+++ b/data.cpp
@@ -868,7 +868,8 @@ void DataManager::SetDefaultValues()
 	mConst.SetValue("tw_mtp_enabled", "0");
 #endif
 	mPersist.SetValue("tw_mount_system_ro", "2");
-	mPersist.SetValue("tw_never_show_system_ro_page", "0");
+	//mPersist.SetValue("tw_never_show_system_ro_page", "0");
+	mPersist.SetValue("tw_never_show_system_ro_page", "1");
 	mPersist.SetValue("tw_language", EXPAND(TW_DEFAULT_LANGUAGE));
 	LOGINFO("LANG: %s\n", EXPAND(TW_DEFAULT_LANGUAGE));
 
@@ -890,9 +891,9 @@ void DataManager::SetDefaultValues()
 	mConst.SetValue("tw_oem_build", "1");
 #else
 	mConst.SetValue("tw_oem_build", "0");
-	mPersist.SetValue("tw_app_prompt", "1");
-	mPersist.SetValue("tw_app_install_system", "1");
-	mData.SetValue("tw_app_install_status", "0"); // 0 = no status, 1 = not installed, 2 = already installed
+	//mPersist.SetValue("tw_app_prompt", "1");
+	//mPersist.SetValue("tw_app_install_system", "1");
+	//mData.SetValue("tw_app_install_status", "0"); // 0 = no status, 1 = not installed, 2 = already installed
 #endif
 
         mData.SetValue("tw_enable_adb_backup", "0");
